plugin.rb

```diff
@@ -122,6 +122,11 @@
     register_editable_user_custom_field(field)
   end
 
+  on(:post_alerter_before_post) do |post, new_record, notified|
```

Comment:
Would it be safer to check whether the `post_alerter_before_post` event exists in the list of events so we can subscribe only to the one event we are certain exists?

Reply:
There is no way of checking that because the events are triggered dynamically and none of them are registered in an array or similar data structure.

Reply:
RIght, sorry. I misunderstood how `DiscourseEvent.events` worked.